logger/transport.go

```diff
@@ -98,6 +98,12 @@ func (t *transport) send(msgs []Message) error {
 			Level: msg.Options.Level,
 		}
 
+		timestamp := msg.Options.Timestamp
+		if timestamp.IsZero() {
+			timestamp = time.Now()
+		}
+		line.Timestamp = timestamp.UnixNano() / int64(time.Millisecond)
```

Comment:
I was also going to originally mention that this logic is a bit confusing to read at first. It's mostly an issue with the variable names not providing enough context. My understanding is that the constant `time.Millisecond` is the `Duration` of 1ms. `Duration` is measured in nanoseconds so `time.Millisecond` is the number of nanoseconds in 1 millisecond. That's why the math above works.

Might be worthwhile to drop a comment about it. Not necessary, gonna approve regardless.